### PR TITLE
feat: product-page-done

### PR DESCRIPTION
**Pull request by ankit9015**
Product page for ecommerce 

https://user-images.githubusercontent.com/67504902/154898500-01b23840-9b08-414e-8ee9-b733c03bf185.mp4

### CODE REVIEW

File: product-page.css [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+.badge-wishlist {
+    width: 3rem;
+    height: 3rem;
+    border-radius: 50%;
+    background-color: var(--pure-white);
+    position: absolute;
+    top: 1rem;
+    right: 1rem;
+    /* transform: translate(50%, -50%);   */
+    border: 2px solid black;
```
Comment (Priya-Patel31) [inline product-page.css:70]: fix : use variables for using colors rather than using them directly

File: product-page.css [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+.badge-wishlist {
+    width: 3rem;
+    height: 3rem;
+    border-radius: 50%;
+    background-color: var(--pure-white);
+    position: absolute;
+    top: 1rem;
+    right: 1rem;
+    /* transform: translate(50%, -50%);   */
```
Comment (Priya-Patel31) [inline product-page.css:69]: refactor : remove commented code

File: product-page.css [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+    border: 2px solid black; 
+    justify-content: center;
+    align-items: center;
+}
+
+
```
Comment (Priya-Patel31) [inline product-page.css:75]: refactor : remove these unnecessary white spaces

File: product-page.html [hunk 4]
```diff
@@ -0,0 +1,503 @@
...
+    <main class="flex-row product-page-main text-md">
+        <aside class="filter-column flex-column">
+            <div class="flex-row main-space-between">
+                <input type="submit" class="text-lg small-button button-secondary" value="Filter">
+                <input type="submit" class="text-lg small-button button-outline-secondary" value="Clear">
+            </div>
+            <div>
+                <h2 class="text-lg font-bold">Price</h2>
+                <div class="text-md price-filter-container flex-row">
+                    <span class="price-filter-min text-gray text-lg">50</span>
```
Comment (Priya-Patel31) [inline product-page.html:77]: fix : avoid using non-semantic tags i.e span

File: product-page.css [hunk 5]
```diff
@@ -0,0 +1,97 @@
+.product-page-main {
+    font-size: 1.6rem;
+}
+
+.filter-column {
+    background-color: var(--white);
+    align-self: stretch;
+    gap: 1rem;
+    min-width: 300px;
```
Comment (Priya-Patel31) [inline product-page.css:9]: refactor : give these values in rem.

**Review by arjundevn — APPROVED** [review]
Looks good

File: product-page.html [hunk 6]
```diff
@@ -0,0 +1,503 @@
...
+                  <div class="card-vertical">
+                    <img class="card-img" src="./images/product-card.png" alt="card image" />
+                    <div class="badge-wishlist flex-row">
+                        <i class="fa fa-heart wishlist-icon" aria-hidden="true"></i>
+                    </div>
+                    <div class="card-body">
+                      <h1 class="card-title">Men Premium Jacket</h1>
+
+                      <h6 class="card-title-sub text-gray text-lg">50%- off</h6>
+                      <p class="card-description text-center"><span class="price-new text-lg font-extrabold">₹2000  </span> <span class="text-lg font-light price-old font-light text-gray">₹3999</span></p>
```
Comment (Priya-Patel31) [inline product-page.html:222]: fix : avoid using non-semantic tags

**Review by kotalwar-krishna — APPROVED** [review]
looks good

File: product-page.css [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+.line-height-s {
+    line-height: 1.2;
+}
+
+.main-space-between {
+    justify-content: space-between;
+}
+
+.cross-space-between {
+    align-items: space-between;
```
Comment (Priya-Patel31) [inline product-page.css:37]: align-items : space-between I don't think so we have any property like this . So make correction in it

**Review by Priya-Patel31 — APPROVED** [review]
There are few changes which you should change it as they are necessary . Rest everything seems fine to me.

**Pull request merged**